docs/strategy_analysis_example.md

```diff
@@ -107,6 +107,22 @@ trades = load_trades_from_db("sqlite:///tradesv3.sqlite")
 trades.groupby("pair")["sell_reason"].value_counts()
 ```
 
+## Analyze the loaded trades for trade parallelism
+This can be useful to find the best `max_open_trades` parameter, when used with backtesting in conjunction with `--disable-max-market-positions`.
+
+`parallel_trade_analysis()` returns a timeseries dataframe with an "open_trades" column, specifying the number of open trades for each candle.
```

Comment:
seems like there is no analyze_trade_parallelism() you call it analyze_trade_parallelism() ?

Reply:
nice catch - i renamed the function last-minute since i found `analyze_trade_parallelism()` to be easier to grasp ... but apparently forgot to swap it everywhere ...